ql/experimental/callablebonds/treecallablebondengine.cpp

```diff
@@ -71,12 +76,22 @@ namespace QuantLib {
             lattice = model_->tree(timeGrid);
         }
 
+        OneFactorModel::ShortRateTree *sr=
```

Comment:
This only need to be done if the spread is not 0.0.  I'd check that before casting.